Assignment1/assignment2/lib/assignment2.dart

```diff
@@ -0,0 +1,74 @@
+import 'package:assignment2/utilities.dart' as utilities;
+import 'package:assignment2/users.dart' as users;
+import 'dart:io';
+import 'dart:collection';
+
+void addUser() {
+  try {
+    print('Enter Full Name :');
+    String name = stdin.readLineSync()!;
+    utilities.nameHandler(name);
+    print('Enter Age :');
+    int age = int.parse(stdin.readLineSync()!);
+    utilities.ensurePositive(age);
+    print('Enter Address :');
+    String address = stdin.readLineSync()!;
+    print('Enter Roll Number :');
+    int rollNumber = int.parse(stdin.readLineSync()!);
+    utilities.ensurePositive(rollNumber);
+    utilities.ensureRollNumber(rollNumber);
```

Comment:
All the above three lines that validate the roll number can be moved to a rollNumberValidator method.